asciidoc-confluence-publisher-converter/src/main/java/org/sahli/asciidoc/confluence/publisher/converter/NoOpPageTitlePostProcessor.java

```diff
@@ -0,0 +1,13 @@
+package org.sahli.asciidoc.confluence.publisher.converter;
```

Comment:
License is missing.

Reply:
This is true, but not only for new classes in this pull request - in previous pull requests, new classes and interfaces also had no license headers (e.g. `AsciidocPagesStructureProvider`). I see two options:

a) we add the license header with every new file
b) we add (or update) all missing or outdated license headers as part of the release routine

Any preferences?